rules-topics/10_magic-skills.md

```diff
@@ -680,7 +680,7 @@ Characters may alternatively choose to explicitly memorize their spell list. Doi
 | **Level** | 7 |
 | **Duration**  | 20 seconds |
 | **Incant**  | "By nature, I confine you." |
-| **Description** |  This spell renders the target completely unable to move while under the effects of this spell.  This is a CONFINE effect. |
+| **Description** |  This spell renders the target completely unable to move while under the effects of this spell. They also may not activate any game skills other than Escape Artist. This is a CONFINE effect. |
```

Comment:
I would drop this. There are other abilities that can be used while CONFINED, but I don't think we want to explicitly list them here. (Rip from, Minimize,...)